From 2905fc861acda3d134a198e56ef2f6c962ad3061 Mon Sep 17 00:00:00 2001
From: Daniel Boles <dboles.src@gmail.com>
Date: Wed, 12 Dec 2018 19:03:11 +0000
Subject: [PATCH] Revert "Fix deprecation warnings"

This reverts commit 5aedfe048b0fe07382433f1a78ec5bd26acd82dd.

It had a typo that broke the build, only replaced half of the uses, and
replaced them with other functions that are also deprecated anyway.
---
 docs/tools/shooter.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/tools/shooter.c b/docs/tools/shooter.c
index 3d910c6ab8..8cefa8bbf7 100644
--- a/docs/tools/shooter.c
+++ b/docs/tools/shooter.c
@@ -170,10 +170,10 @@ take_window_shot (Window         child,
       y_orig = 0;
     }
 
-  if (x_orig + width > gdk_screen_get_width (gdk_screen_get_dfeault ()))
+  if (x_orig + width > gdk_screen_width ())
     width = gdk_screen_width () - x_orig;
 
-  if (y_orig + height > gdk_screen_get_height (gdk_screen_get_default ()))
+  if (y_orig + height > gdk_screen_height ())
     height = gdk_screen_height () - y_orig;
 
   tmp = gdk_pixbuf_get_from_window (window,
-- 
2.19.2

